components/layout/wrapper.rs

```diff
@@ -120,6 +120,20 @@ impl<'ln> ServoLayoutNode<'ln> {
             chain: self.chain,
         }
     }
+
+    pub fn initialize_data(self) {
+        let has_data = unsafe { self.borrow_data_unchecked().is_some() };
+        if !has_data {
```

Comment:
I'd argue that using `is_none` is better here, I don't see much point of using `if let` when the block only gets run if `self.borrow_data_unchecked()` returns `None`.